ztp/gitops-subscriptions/argocd/example/policygentemplates/hub-side-templating/group-du-sno-ranGen-templated.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     - fileName: DisableOLMPprof.yaml # wave 10
       policyName: "gr-du-sno-cfg-pc-templated"
     ###
-    - fileName: ClusterLogging.yaml # wave 10
+    - fileName: ClusterLogForwarder.yaml # wave 10
```

Comment:
This looks to duplicate the inclusion of log forwarder on line 25. I think we just need to delete the ClusterLogging CR?

Reply:
I've dropped this duplicate